static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -24,15 +27,48 @@ type Installation = {
 type GithubInstallationProps = {
   has_scm_multi_org: boolean;
   installation_info: Installation[];
+  organization: Organization;
   organization_slug: string;
 };
 
+const renderInstallationButton = ({
```

Comment:
Since this is the 'default' component to be shown I assume the user will always be self hosted if they're seeing this component.